thoth/common/logging.py

```diff
@@ -148,6 +148,19 @@ def _get_sentry_integrations() -> List[object]:
 
     return integrations
 
+def before_send_handler(event, hint):
+    """ Ignores the exceptions passed in as as the environment variable in a comma separated manner.
+    """
+    ignored_exceptions = os.environ("THOTH_SENTRY_IGNORE_EXCEPTION")
+    print(ignored_exceptions)
+    if ignored_exceptions:
+        if 'exc_info' in hint:
+            exc_type, exc_value, tb = hint['exc_info']
+        for exception in ignored_exceptions.split(','):
+            if isinstance(exc_value, exception):
```

Comment:
`exception` will be of type string (not type, but the value of type). What is the type of exc_value?

Reply:
`{'log_record': <LogRecord: connexion.decorators.validation, 40, /home/sai/.local/share/virtualenvs/user-api-ZDh_dRcv/lib/python3.6/site-packages/connexion/decorators/validation.py, 204, "http://localhost:5000/api/v1/advise/python?recommendation_type=stable validation error: 3 is not of type 'string' - 'runtime_environment.python_version'">}`
This is what the hint value looks like, its of the type - 'dict'

Reply:
I was referring to variable `exception` which you use in the loop to iterate over ignored exceptions.

What about the `event` passed? What information does it carry?

Reply:
In the current method, `exception` is the variable that iterates over the exceptions to be ignored. And the event has various details used by sentry, like stack trace, packages installed, error name. (link to a sample event - https://paste.ofcode.org/3WMTFq9zMLzTyku29EMutR
api called event - https://pastecode.xyz/view/32f334d2)